.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,14 +8,14 @@ https://git.k8s.io/community/contributors/devel/sig-release/release.md#issuepr-k
 -->
 
 **What type of PR is this?**
-> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
->
-> /kind bug
-> /kind cleanup
-> /kind deprecation
-> /kind design
-> /kind documentation
-> /kind feature
+<!-- Uncomment only one ` /kind <>` line: -->
+
+<!-- /kind bug -->
```

Comment:
Do we still need to remove leading whitespace when this is uncommented?

Reply:
Looks like it.
General note: We may not want to do this.
While it feels more syntactically correct, the contributor experience is now you have to delete from the front AND back of each line.

Maybe this instead?

```markdown
<!--
/kind bug
/kind cleanup
/kind deprecation
/kind design
/kind documentation
/kind feature
-->
```

Reply:
AHA!
Nevermind, that won't work.

If the bot command is at the beginning of the line, it'll execute, even if ensconced in a comment tag. That was the reason for the leading whitespace.

Reply:
> Maybe this instead?
> 
> ```gfm
> <!--
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind feature
> -->
> ```

This would make the general workflow even more simpler, because I now can just copy paste one line from above below the whole comment. :+1:

Reply:
Updated the PR with another proposal.

Reply:
@saschagrunert -- The new suggestion won't work. See my comment above: https://github.com/kubernetes/release/pull/1076#discussion_r379877971

Reply:
^^ if you look at the history on this PR, you'll see that Prow ended up adding all of the kind labels because of this.

Reply:
Yeah now I got it.